src/components/LandingPage/HeroBanner/HeroBanner.component.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+    ORGANISATIONS,
+  } from '@/constants/organisations';
+  import { paths } from '@/constants/paths';
+  import Image from 'next/image';
+  import Link from 'next/link';
+
+export const HeroBanner = () => {
+    return (
+        <>
+            <div className = "bg-gradient-to-r from-verydarkblue to-darkblue rounded-[32px] min-h-[312px] text-center max-w-[1168px]">
+                <div className='text-[40px] text-white h-[248px] leading-[60px] font-bold min-h-[120px] place-content-center'>
+                    <h1>Budujemy liderów i liderki przyszłości</h1>
+                </div>
+                <hr className='border-[#383838]'/>
+                <div className="flex gap-2 mix-blend-luminosity flex-wrap place-content-center min-h-[64px] p-[24px]">
```

Comment:
```suggestion
                <div className="flex gap-4 mix-blend-luminosity flex-wrap place-content-center min-h-[64px] p-[24px]">
```

`gap` na designie wynosi 16px co odpowiada 4 w tailwind